app/controllers/v0/ppiu_controller.rb

```diff
@@ -14,6 +14,8 @@ def update
       pay_info = EVSS::PPIU::PaymentAccount.new(ppiu_params)
       raise Common::Exceptions::ValidationErrors, pay_info unless pay_info.valid?
       response = service.update_payment_information(pay_info)
+      email = current_user.vet360_contact_info&.email || current_user.email
+      DirectDepositMailer.build(email, params[:ga_client_id]).deliver_now if email
```

Comment:
I haven't followed all the emailer conversation threads but are we allowing direct deposit to be updated if the user doesn't have an email available? If so, do we want to log when we update DD without an email going out?

Reply:
> log when we update DD without an email going out
What would this look like?

Reply:
Far be it from me from cluttering sentry, but we could send out an `info` level prompt. But only if its actually of any use to us. If anything it may be worth discussing NOT allowing an update without a valid email in place...